GOVERNANCE.md

```diff
@@ -0,0 +1,81 @@
+---
+permalink: /governance/
+---
+# Governance
+
+This project uses [Martha's Rules][marthas-rules] for making decisions.  The
+first section below describes these rules; the second lists voting members of
+the project and explains how to become one, while the third lists decisions made
+to date.
+
+[marthas-rules]: https://journals.sagepub.com/doi/10.1177/088610998600100206
+
+## Making Decisions
+
+1.  Before each meeting, anyone who wishes may sponsor a proposal by filing an
+    issue in the GitHub repository tagged "proposal".  Proposals must be filed
+    at least 24 hours before a meeting in order to be considered at that
+    meeting, and must include:
+    -   a one-line summary (the subject line of the issue)
+    -   the full text of the proposal
+    -   any required background information
+    -   pros and cons
+    -   possible alternatives
+
+2.  A quorum is established in a meeting if half or more of voting members are
+    present.
+
+3.  Once a person has sponsored a proposal, they are responsible for it.  The
+    group may not discuss or vote on the issue unless the sponsor or their
+    delegate is present.  The sponsor is also responsible for presenting the
+    item to the group.
+
+4.  After the sponsor presents the proposal, a sense vote is cast for the
+    proposal prior to any discussion:
+    -   Who likes the proposal?
+    -   Who can live with the proposal?
+    -   Who is uncomfortable with the proposal?
+
+5.  If all of the group likes or can live with the proposal, it is immediately
+    accepted with no further discussion.
+
+6.  If most of the group is uncomfortable with the proposal, it is postponed for
+    further rework by the sponsor.
+
+7.  If any members are uncomfortable they can briefly state their objections.  A
+    timer is then set for a brief discussion moderated by the facilitator.
+    After 10 minutes or when no one has anything further to add (whichever comes
```

Comment:
I think this is a good standard for our current group size, but the time limit here does present a barrier for those who are not able to communicate fluently (e.g. those who have disabilities related to verbal communication and/or are not relatively comfortable speaking in English). I would suggest to use a modification from the text for Martha's rules of something like "during roll call, members will state how much time they reasonably require for comments. For example, an abled English speaker may request 2 minutes as reasonable." This way, we can be equitable for any members who require more time to speak.